### PR TITLE
PMM-12375 db ver details inventory services

### PR DESCRIPTION
**Pull request by rasika-chivate**
https://jira.percona.com/browse/PMM-12375

### CODE REVIEW

File: docs/details/dashboards/dashboard-inventory.md [hunk 1]
```diff
@@ -13,25 +13,40 @@ Inventory objects form a hierarchy with Node at the top, then Service and Agents
...
 
-**Service name** - The name or identifier associated with the service being monitored.
+| **Column Name**| **Description**|
+|--------------|--------------------------------|
+| Service name|The name or identifier associated with the service being monitored.|                                        
+| Node name | Name or identifier associated with a specific node.| 
+| Monitoring status| The **Monitoring** column summarizes the status of all the Agents assigned to the service.             | 
+|  Address         | The IP address or DNS where the service is currently running. |
+|  Port         | The port number on which the service is running. |
+|  QAN         | You can check Query Analytics information and the Service Overview Dashboard for each service by clicking on the <image src="../../_images/dots-three-vertical.ico" width="15px" aria-label="triple dots"/> icon in the **Options** column.|
```
Comment (ademidoff) [inline docs/details/dashboards/dashboard-inventory.md:23]: I'm afraid, we do not feature a column called `QAN`. However, if you meant the menu item available when the user clicks on the arrow, then we should mention it outside of this table, which only lists columns.

File: docs/details/dashboards/dashboard-inventory.md [hunk 2]
```diff
@@ -13,25 +13,40 @@ Inventory objects form a hierarchy with Node at the top, then Service and Agents
...
+|  Additional information | You can also check additional information about the service, by clicking on the <image src="../../_images/arrow-downward.ico" width="15px" aria-label="downward arrow"/> icon. This expands the service entry to show reference information like service labels and IDs. |
```
Comment (ademidoff) [inline docs/details/dashboards/dashboard-inventory.md:24]: I understand, this column is called 'Options', right?

File: docs/details/dashboards/dashboard-inventory.md [hunk 3]
```diff
@@ -13,25 +13,40 @@ Inventory objects form a hierarchy with Node at the top, then Service and Agents
...
+#### Attributes
 
-**Port** - The port number on which the service is running.
+These are some of the atributes for a service:
 
-You can check Query Analytics information and the Service Overview Dashboard for each service by clicking on the <image src="../../_images/dots-three-vertical.ico" width="15px" aria-label="triple dots"/> icon in the **Options** column.
+- Each instance of a service gets a `service_type` attribute so one can clearly tell what type of database it is, for instance: `mysql`, `postgresql`, `mongodb`, etc. 
 
-From here you can also check additional information about the service, by clicking on the <image src="../../_images/arrow-downward.ico" width="15px" aria-label="downward arrow"/> icon. This expands the service entry to show reference information like service labels and IDs.
+- Every service is related to a certain node via its `node_id` attribute. This feature allows to support multiple instances on a single node, with different service names, e.g. `mysql1-3306`, and `mysql1-3307`.
```
Comment (ademidoff) [inline docs/details/dashboards/dashboard-inventory.md:35]:
```suggestion
- Every service is related to a certain node via its `node_id` attribute. This feature allows to support monitoring of multiple instances on a single node, with different service names, e.g. `mysql1-3306`, and `mysql1-3307`.
```

File: docs/details/dashboards/dashboard-inventory.md [hunk 4]
```diff
@@ -13,25 +13,40 @@ Inventory objects form a hierarchy with Node at the top, then Service and Agents
...
 
-![!image](../../_images/PMM_Inventory_Service_Selection.png)
+- Starting with PMM 2.41.0, each instance of a service gets a `version` attribute to the response of the endpoint that provides a list of services being monitored by PMM. This makes it easy to visualize the database server version.
+
+    However, following are the imitations:
+    
+    - The version is not captured for the internal PostgreSQL database (currently version 14.9).
```
Comment (ademidoff) [inline docs/details/dashboards/dashboard-inventory.md:41]:
```suggestion
    - The version is not captured for the internal PostgreSQL database.
```

**Review by ademidoff — CHANGES_REQUESTED** [review]

File: docs/details/dashboards/dashboard-inventory.md (hunk 4 above)
Comment (ademidoff) [reply docs/details/dashboards/dashboard-inventory.md:41]: I'd remove the version from here so we don't have to remember to update it later on.

File: docs/details/dashboards/dashboard-inventory.md (hunk 2 above)
Comment (rasika-chivate) [reply docs/details/dashboards/dashboard-inventory.md]: Fixed.

**Review by ademidoff — APPROVED** [review]

**Pull request merged**